pkg/virtctl/root.go

```diff
@@ -36,6 +36,29 @@ func NewVirtctlCommand() *cobra.Command {
 			fmt.Fprint(cmd.OutOrStderr(), cmd.UsageString())
 		},
 	}
+	completionCmd := &cobra.Command{
+		Use:   "completion",
+		Short: "Generates completion scripts for the specified shell",
+		Args:  cobra.MaximumNArgs(1),
+		Long: `To load completion run
+. <(virtctl completion (SHELL TYPE))
+
+To configure bash shell to load completions for each session add to bashrc
+
+# ~/.bashrc or ~/.profile
+. <(virtctl completion (SHELL TYPE))
+`,
+		Run: func(cmd *cobra.Command, args []string) {
+			switch {
+			case len(args) == 0:
+				rootCmd.GenBashCompletion(os.Stdout)
+			case args[0] == "bash":
+				rootCmd.GenBashCompletion(os.Stdout)
+			default:
```

Comment:
I see that that kubectl does some crazy stuff for zsh (see link above).... but there also is a `GenZshCompletion` function.  Maybe worth a try?

Reply:
`GenZshCompletion` generates a similar output than GenBashCompletion
I'll take a look to zsh completion but I think that the "crazy stuff" would be also required here :(

Reply:
ok, understand. Would be awesome if you can add zsh, but if you prefer to skip it for this PR, that's also fine :)